### DISCUSSION TITLE
Mirrored screen not loading

### DISCUSSION THREAD

**Discussion by ecotechie** [srevinsaju/guiscrcpy#395] [General]
scrcpy loading fine, but guiscrcpy not loading mirror window. I can navigate it with the gui, but the screen does not show on desktop.

**To Reproduce**
1. Start guiscrcpy

**Expected behavior**
Mirror window would open, like it does with scrpy

**Screenshots**
![image](https://github.com/srevinsaju/guiscrcpy/assets/11607383/f8169233-49d0-47fc-817c-f7a7425de4f7)

**Desktop (please complete the following information):**

* [ ] Linux
![image](https://github.com/srevinsaju/guiscrcpy/assets/11607383/ecf54001-af70-4401-80b2-d717c934a8b4)

**Additional context**
I'm connecting with wifi debug, which seems to work fine on scrpy and also here. But the screen/mirror window is not loading.

**Comment by ecotechie** [comment]
I installed scrcpy from source and guiscrcpy on snap...

**Comment by ecotechie** [comment]
The `GUISCRCPY_DEBUG=true guiscrcpy` output:

![image](https://github.com/srevinsaju/guiscrcpy/assets/11607383/f151f87a-c1f1-45d6-a1a4-8f611a5ce416)

**Comment by viarotel** [comment]
have a try [Escrcpy](https://github.com/viarotel-org/escrcpy)?

**Comment by cociweb** [comment]
Try the AppImage! Snap does not load it me neither!
